src/components/RecipeModal/RecipeModal.jsx

```diff
@@ -0,0 +1,46 @@
+import "./RecipeModal.css";
+import { useState, useEffect } from "react";
+import * as api from "../../utils/api";
+
+function RecipeModal({ isOpen, onClose, recipeId }) {
+  const [recipeSummary, setRecipeSummary] = useState(null);
+
+  useEffect(() => {
+    const fetchRecipeSummary = async () => {
+      try {
+        if (recipeId) {
+          const summary = await api.getRecipeSummary(recipeId);
+          console.log(summary);
+          setRecipeSummary(summary);
+        }
+      } catch (error) {
+        console.error(error);
+      }
+    };
+    fetchRecipeSummary();
+    console.log(recipeId);
+  }, [recipeId]);
+
+  return (
+    <>
```

Comment:
`<>` empty tags are not needed if you have a common tag that wraps the whole markup of the component